config/locales/en.yml

```diff
@@ -214,6 +214,10 @@ en:
   timeout:
     heading: You're about to be signed out
     body: Your progress through the training has been saved. Other changes made to your account or learning log will not be saved.
+    noscript: |
+      For security, you'll be signed out after %{timeout_duration} minutes of activity. You'll be signed out at %{sign_out_time}.
+
```

Comment:
should this be `%{timeout_duration} minutes of **inactivity**`

Reply:
D'oh. Yes. Good spot